# deep-learning-SC-ST-papers

This repository keeps track of the latest papers on single-cell and spatial transcriptomics analysis with deep learning methods. We categorize them based on individual tasks.

- [Spatial Transcriptomics](#spatial-transcriptomics)
- [Perturbation](#perturbation)

## Spatial Transcriptomics
1. [2022 Nature Methods] **Museum of spatial transcriptomics** [[paper]](https://www.nature.com/articles/s41592-022-01409-2)

## Database
1. [2024 Nature Methods] **SpatialData: an open and universal data framework for spatial omics**[[paper]](https://www.nature.com/articles/s41592-024-02212-x)

## Perturbation
1. [2025 biorxiv] **scCausalVI disentangles single-cell perturbation responses with causality-aware generative model** [[paper]](https://www.biorxiv.org/content/10.1101/2025.02.02.636136v1) [[code]](https://github.com/ShaokunAn/scCausalVI)

2. [2024 Nature Biotech] **biolord: Disentanglement of single-cell data with biolord** [[paper]](https://www.nature.com/articles/s41587-023-02079-x?fromPaywallRec=true) [[code]](https://github.com/nitzanlab/biolord)

3. [2024 Nature Communication] **scDisInFact: disentangled learning for integration and prediction of multi-batch multi-condition single-cell RNA-sequencing data** [[paper]](https://www.nature.com/articles/s41467-024-45227-w)[[code]](https://github.com/ZhangLabGT/scDisInFact)

4. [2024 Cell Repost Medicine] **scRank: scRank infers drug-responsive cell types from untreated scRNA-seq data using a target-perturbed gene regulatory network** [[paper]](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(24)00260-X)[[code]](https://github.com/ZJUFanLab/scRank)

5. [2024 Bioinformatics] **Predicting single-cell cellular responses to perturbations using cycle consistency learning** [[paper]](https://academic.oup.com/bioinformatics/article/40/Supplement_1/i462/7700878)[[code]](https://github.com/hliulab/cycleCDR)

6. [2023 Nature Methods] **CINEMA-OT: Causal identification of single-cell experimental perturbation effects with CINEMA-OT** [[paper]](https://www.nature.com/articles/s41592-023-02040-5) [[code]](https://github.com/vandijklab/CINEMA-OT)

7. [2023 Natuer Methods] **contrastiveVI: Isolating salient variations of interest in single-cell data with contrastiveVI** [[paper]](https://www.nature.com/articles/s41592-023-01955-3)[[code]](https://github.com/suinleelab/contrastiveVI-reproducibility)

8. [2023 Molecular Systems Biology] **CPA: Predicting cellular responses to complex perturbations in high‚Äêthroughput screens** [[paper]](https://www.embopress.org/doi/full/10.15252/msb.202211517)[[code]](https://github.com/theislab/cpa)

9. [2022 NeurIPS]**ChemCPA: Predicting Cellular Responses to Novel Drug Perturbations at a Single-Cell Resolution** [[paper]](https://neurips.cc/virtual/2022/poster/53227)[[code]](https://github.com/theislab/chemCPA)

10. [2019 Nature Methods] **scGen predicts single-cell perturbation responses** [[paper]](https://www.nature.com/articles/s41592-019-0494-8)[[code]](https://github.com/theislab/scgen)





